docs/admin/access.rst

```diff
@@ -136,13 +139,13 @@ Users, roles, groups and permissions
 The authentication models consist of several objects:
 
 `Permission`
-    Individual permissions defined by Weblate. You can not assign individual
-    permissions, this can only be done through assignment of roles.
+    Individual permissions defined by Weblate. Permissions can not be
+    assigned to single users. This can only be done through assignment of roles.
 `Role`
-    Role defines a set of permissions. This allows reuse of these sets in
-    several places, and makes the administration easier.
+    A Role defines a set of permissions. This allows reuse of these sets in
+    several places, easing administration.
 `User`
-    Users can be members of several groups.
+    Users can belong to several groups.
 `Group`
     Groups connect roles, users and authentication objects (projects,
```

Comment:
I sense this can be improved.

Reply:
once you know, make a suggestion. I can’t come up with anything better.